theano/gof/op.py

```diff
@@ -694,6 +694,9 @@ def __ne__(self, other):
     # Python implementation #
     #########################
 
+    def grad2(self, inputs, outputs, output_grads):
```

Comment:
Since `Op.Lop` does not actually exist, and that `theano.gradient.Lop` actually calls `theano.grad`, I would favor that one.
But I have no real objection against `grad_chain`, except it is less descriptive.